### PR TITLE
fix(route): 豆瓣榜单问题

### PR DESCRIPTION
**Pull request by xuanqb**
## 完整路由地址 / Example for the proposed route(s)

```routes
/douban/movie/weekly/movie_top250
/douban/movie/weekly/subject_real_time_hotest
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ ] 新的路由 New Route
  - [ ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ ] 文档说明 Documentation
  - [ ] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/douban/other/weekly_best.js [hunk 1]
```diff
@@ -28,11 +28,13 @@ module.exports = async (ctx) => {
         link: `https://m.douban.com/subject_collection/${type}`,
         description: infoResponse.data.description,
 
-        item: data.map(({ title, info, cover, url, rating, year, release_date, null_rating_reason, description }) => {
+        item: data.map(({ title, info, cover, cover_url, url, rating, year, release_date, null_rating_reason, description }) => {
```
Comment (TonyRL) [inline lib/v2/douban/other/weekly_best.js:31]:
```suggestion
        item: data.map(({ title, info, cover_url, url, rating, year, release_date, null_rating_reason, description }) => {
```

File: lib/v2/douban/other/weekly_best.js [hunk 2]
```diff
@@ -28,11 +28,13 @@ module.exports = async (ctx) => {
...
             const release = `${year}.${release_date}`;
             const rate = rating ? `${rating.value.toFixed(1)}分` : null_rating_reason;
-
-            description = `标题：${title} <br> 影片信息：${info} <br> 上映日期：${release} <br> 评分：${rate} <br> <img src="${cover.url}"> <p>${description}</p>`;
+            if (cover && cover.url) {
+                cover_url = cover.url;
+            }
+            description = `标题：${title} <br> 影片信息：${info} <br> 上映日期：${release} <br> 评分：${rate} <br> <img src="${cover_url}"> <p>${description}</p>`;
```
Comment (TonyRL) [inline lib/v2/douban/other/weekly_best.js:37]:
```suggestion

            description = `标题：${title} <br> 影片信息：${info} <br> 上映日期：${release} <br> 评分：${rate} <br> <img src="${cover_url}"> <p>${description}</p>`;
```

**Review by TonyRL — COMMENTED** [review]
I think simply replace `cover.url` with `cover_url` is enough as I can't see any `cover` property in the response

File: lib/v2/douban/other/weekly_best.js (hunk 1 above)
Comment (xuanqb) [reply lib/v2/douban/other/weekly_best.js:31]: the cover property is useful in the '`douban/movie/weekly/subject_real_time_hotest `' interface

**Comment by xuanqb** [comment]
the cover property is useful in the 'douban/movie/weekly/subject_real_time_hotest ' interface

**Pull request merged**